macros/detect_column_changes.sql

```diff
@@ -0,0 +1,41 @@
+{% macro detect_column_changes(a_relation, b_relation, primary_key, exclude_columns=[]) %}
+
+{% set column_names = dbt_utils.get_filtered_columns_in_relation(from=a_relation, except=exclude_columns) %}
```

Comment:
what if there are columns in a that are not in b or vice versa? do you just have to specify those in your "exclude_columns" field?

Reply:
i think that's the way the other macros have worked, yeah -- this line was grabbed from compare_relations

Reply:
Ok great - I like that flexibility!